terra-env/main.tf

```diff
@@ -72,7 +72,7 @@ module "sam" {
 }
 
 module "workspace_manager" {
-  source = "github.com/broadinstitute/terraform-ap-modules.git//terra-workspace-manager?ref=terra-workspace-manager-0.7.3"
+  source = "github.com/broadinstitute/terraform-ap-modules.git//terra-workspace-manager?ref=jaycarlton-token-creator"
```

Comment:
I'm not conversant in TF. Why does this have your name in it? Is it temporary? Does it need versioning?

Reply:
It's temporary. It will be replaced, after approval, by a new tag value.